src/widgets/search-box/search-box.js

```diff
@@ -124,6 +131,27 @@ const renderer = ({
   }
 };
 
+function renderAfterInit({
+  containerNode,
+  query,
+  loadingIndicator,
+  isSearchStalled,
+}) {
+  const input = getInput(containerNode);
+  const isFocused = document.activeElement === input;
+  if (!isFocused && query !== input.value) {
+    input.value = query;
+  }
+
+  if (loadingIndicator) {
+    if (isSearchStalled) {
+      containerNode.firstChild.classList.add('stalled-search');
```

Comment:
Should this CSS class name follow ais- conventions? And be documented somewhere?

Reply:
Oh yes sure. Also it doesn't work if you use an input as container :/

Reply:
Maybe document that part, but I won't be surprised as a dev for it not to be supported.